src/mrpro/operators/_SensitivityOp.py

```diff
@@ -25,7 +25,9 @@ def __init__(
         self,
         csm: CsmData,
     ) -> None:
-        self.C = csm.data
+
+        super().__init__()
+        self.register_buffer('CSM', csm)
```

Comment:
To be somewhat consistent the buffer should be 'csm' and not capitalised. Maybe even '_csm' as we want to discourage users from overwriting it after the _init_

Reply:
I would not underscore it, as it is ok to change the cms later on.

Reply:
I was thinking of something like 
```
S = SensitivityOp(csm_cpu)
S.cuda()
S.csm = csm_cpu
```
Will csm then still be a registered_buffer? Will csm then still be on the cuda device?

Reply:
Will csm then still be a registered_buffer? -> Yes. It would replace the value of the existing buffer, but it would still be a buffer.
Will csm then still be on the cuda device? -> Only if you moved it there. The .cuda move moves all buffers, it does not enforce that they are kept there